frontend/src/store/controller/reducers/backend.ts

```diff
@@ -128,7 +135,12 @@ export const eventLogReducer = (
     }
     case BACKEND_CONNECTION_DOWN: {
       // Make an ephemeral frontend-only event
-      const logEvent = (action.update as unknown) as LogEvent;
+      const update = {
+        code: LogEventCode.frontend_backend_connection_down,
+        type: LogEventType.system,
+        time: new Date().getTime(),
+      };
+      const logEvent = (update as unknown) as LogEvent;
```

Comment:
Can we get rid of this typecast now that we're explicitly constructing `update`?